exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -727,10 +727,6 @@ func (b *Bitfinex) GetDepositAddress(c currency.Code, accountID string) (string,
 
 // WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is submitted
 func (b *Bitfinex) WithdrawCryptocurrencyFunds(withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
-	if err := withdrawRequest.Validate(); err != nil {
-		return nil, err
-	}
-
```

Comment:
Seems like you are removing this across the board, how come?

Reply:
At the time, I was thinking about how RPCServer/engine was the only caller of withdraw requests and to move the validate to the WithdrawManager. You bringing this up highlighted that someone may just use the packages for API usage etc and not necessarily the whole GCT bot application, so reverting may be a good idea to ensure all requests are validated

Reply:
Might be better to bypass the config checks and just have it withdraw the manager side instead. If they are going to use it as a package.